site/content/en/docs/manual/advanced/task-synchronization.md

```diff
@@ -4,19 +4,21 @@ linkTitle: 'Task synchronization'
 weight: 19
 ---
 
+> Notice: this feature works only if a git repository was specified when the task was created.
+
 1. At the end of the annotation process, a task is synchronized by clicking
-   `Synchronize` on the task page. Notice: this feature
-   works only if a git repository was specified when the task was created.
+   `Synchronized` on the task page.
 
-   ![](/images/image106.jpg)
+   ![](/images/image110.jpg)
 
-1. After synchronization the button `Sync` is highlighted in green. The
-   annotation is now in the repository in a temporary branch.
+1. The annotation is now in the repository in a temporary branch.
+   The next step is to go to the repository and manually create a pull request to the main branch.
 
-   ![](/images/image109.jpg)
+1. After confirming the PR, when the annotation is saved in the main branch,
+   the button changes to `Merged` and is highlighted in green.
 
-1. The next step is to go to the repository and manually create a pull request to the main branch.
+   ![](/images/image109.jpg)
 
-1. After confirming the PR, when the annotation is saved in the main branch, the color of the task changes to blue.
+If synchronization fails, the `Synchronize` button will turn red:
```

Comment:
Are you sure it is true? ``Synchronize`` button is red when annotation in the task does not correspond annotations in the repository. When it corresponds, it is blue.